public/js/game.js

```diff
@@ -90,7 +94,7 @@ function update() {
 
 	const hitPlatform = game.physics.arcade.collide(player, platforms);
 	const starPlatform = game.physics.arcade.collide(stars, platforms);
-	const hitStars = game.physics.arcade.overlap(player, stars, (player, star) => star.kill(), null, this);
+	const hitStars = game.physics.arcade.overlap(player, stars, (player, star) => {star.kill(); score += 10; scoreText.text = 'Score: ' + score;}, null, this);
```

Comment:
I'd actually make this it's own couple of functions.

```
function addToScore(amount) {
	if (Number.isNaN(amount)) {
		return console.log('amount should be a Number');
	}
	
	return score += amount; // Return the value, just in case others would like to use it.
}

function collectStar(player, star) {
	star.kill();
	addToScore(10);
}

...

	const hitStars = game.physics.arcade.overlap(player, stars, collectStar, null, this);
	scoreText.text = `Score: ${score}`; // Don't need to run this from the callback for it to update
```